docs/REFERENCES.md

```diff
@@ -0,0 +1,94 @@
+# Hardware References
+
+This doc will explain what Hardware references are, how to define them, and how to use them in a Template.
+
+## What are References?
+
+The Hardware custom resource defines a field, `spec.references`, which allows for 
+
+The string name under `spec.reference` can be anything and will be used to reference the object in the Template. The name is not required to be the same as the name of the object being referenced.
```

Comment:
which allows for...
(And spec.reference is not a thing)

which provides a list of resources types that can be accessed to retrieve Hardware data. Each reference, must have a string name. It can be anything ...